tests/test_authorization.py

```diff
@@ -1,8 +1,33 @@
+import unittest.mock
+import pytest
+import werkzeug.exceptions
+
 from timeless.access_control.authorization import is_allowed
 from timeless.access_control.methods import Method
+from timeless.access_control.views import SecuredView
 
 
 def test_owner_can_access_location():
-    assert (
-        is_allowed(method=Method.DELETE, resource="location") == True
-    )
\ No newline at end of file
+    assert is_allowed(method=Method.DELETE, resource="location")
+
+
+@pytest.fixture
+def secured_view(app):
+    view = SecuredView()
+    view.resource = "test-resource"
+    with app.test_request_context("/", method="POST"):
+        yield view
+
+
+@unittest.mock.patch("timeless.access_control.authorization.is_allowed")
+def test_secured_view_access_ok(mocked_is_allowed, secured_view, app):
+    mocked_is_allowed.return_value = True
+    with pytest.raises(AssertionError):
```

Comment:
@vryazanov Do we really need `POST` method in fixture? Also it's not clear to understanding why here should be `AssertionError`. May be let's make `GET` request and do not raise any exceptions?  WDYT?

Reply:
@vladarefiev It's made this way because `SecuredView` does not have implemented methods `get`, `post` end etc. If `MethodView` doesn have method it raises `AssertionError`

Reply:
@vladarefiev I'd say it's a generic test for generic view

Reply:
@vryazanov thanks. Now it’s clear